files/zh-cn/web/css/calc/index.md

```diff
@@ -29,7 +29,7 @@ width: calc(100% - 80px);
 
 ### 备注
 
-- **`+` 和 `-` 运算符的两边必须要有空白字符。**比如，`calc(50% -8px)` 会被解析成为一个无效的表达式，解析结果是：一个百分比 后跟一个负数长度值。而加有空白字符的、有效的表达式 `calc(8px + -50%)` 会被解析成为：一个长度 后跟一个加号 再跟一个负百分比。
+- **`+` 和 `-` 运算符的两边必须要有空白字符。** 比如，`calc(50% -8px)` 会被解析成为一个无效的表达式，解析结果是：一个百分比 后跟一个负数长度值。而加有空白字符的、有效的表达式 `calc(8px + -50%)` 会被解析成为：一个长度 后跟一个加号 再跟一个负百分比。
```

Comment:
minor (see: https://github.com/mdn/content/blob/b3e92f9e0724a177977560b51673b391bfdfc5e4/files/en-us/web/css/calc/index.md?plain=1#L41):

```suggestion
- `+` 和 `-` 运算符的**两边必须要有{{Glossary("whitespace", "空白字符")}}**。比如，`calc(50% -8px)` 会被解析成为一个无效的表达式，解析结果是：一个百分比 后跟一个负数长度值。而加有空白字符的、有效的表达式 `calc(8px + -50%)` 会被解析成为：一个长度 后跟一个加号 再跟一个负百分比。
```

Reply:
yeap, didn't check the source first...thx!